docs/release/release-2.0.rst

```diff
@@ -10,6 +10,7 @@ New Features
 * Change the default configuration directory to ``~/.prestoadmin`` and the
   default log directory to ``~/.prestoadmin/log``.
 * Remove the requirement for running and installing presto-admin with sudo.
```

Comment:
How about "The user specified in config.json still needs sudo access on the Presto nodes in order to execute commands like installing the rpm and setting permissions on the configuration files."